### PR TITLE
access the logo from station

### PR DESCRIPTION
**Pull request by Pierregvx**

### CODE REVIEW

File: int/api/utils/utils.go
Comment (Pierregvx) [inline int/api/utils/utils.go:1]: I created a package utils to avoid circular imports

File: api/swagger/server/restapi/resource/swagger.yml [hunk 1]
```diff
@@ -610,6 +610,45 @@ paths:
...
+          type: string
+          required : true
+          description: Plugin unique identifier.
+      produces:
+        - image/*
+        - application/json
+        - text/javascript
+        - text/html
+        - text/css
+        - text/webp
```
Comment (Thykof) [inline api/swagger/server/restapi/resource/swagger.yml:629]:
```suggestion
```

File: int/api/myplugin/logo.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+type imageResponder struct {
+	payload io.Reader
+}
+
+func (ir *imageResponder) WriteResponse(rw http.ResponseWriter, producer runtime.Producer) {
+	io.Copy(rw, ir.payload)
+}
+
```
Comment (Thykof) [inline int/api/myplugin/logo.go:32]:
```suggestion
```

File: int/api/myplugin/logo.go [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+		return operations.NewPluginManagerGetLogoNotFound()
+	}
+	info := plgn.Information()
+	logoPath := info.Logo
+
+	// Open the logo file
+	logoFile, err := os.Open(logoPath)
+	if err != nil {
+		log.Printf("Error opening logo file: %s", err)
+		return operations.NewPluginManagerGetLogoInternalServerError()
```
Comment (Thykof) [inline int/api/myplugin/logo.go:45]: please add a payload, also bellow

**Review by Thykof — APPROVED** [review]

**Review by gregLibert — APPROVED** [review]

File: api/swagger/server/restapi/resource/swagger.yml [hunk 4]
```diff
@@ -581,6 +581,40 @@ paths:
...
+  /plugin-manager/{id}/logo:
+    get:
+      summary: Get plugin logo
+      operationId: pluginManagerGetLogo
+      parameters:
+        - in: path
+          name: id
+          type: string
+          required : true
+          description: Plugin unique identifier.
```
Comment (peterjah) [inline api/swagger/server/restapi/resource/swagger.yml:593]: update the description!

**Review by peterjah — APPROVED** [review]

**Pull request merged**